OpenSim/Simulation/Wrap/WrapObject.cpp

```diff
@@ -74,6 +74,9 @@ void WrapObject::constructProperties()
 
 const PhysicalFrame& WrapObject::getFrame() const
 {
+    if (!_frame) {
+        OPENSIM_THROW_FRMOBJ(OpenSim::Exception, "Tried to call WrapObject::getFrame before the frame has been set. This can happen if the WrapObject is part of the WrapObjectSet of a PhysicalFrame (see: opensim-core issue #3465)");
```

Comment:
"This can happen if the WrapObject is part of the WrapObjectSet of a PhysicalFrame" might not be as helpful for the user as stating what they should do to fix the problem (e.g., "Make sure xyz method is called to set the frame before doing xyz operation.").